server.js

```diff
@@ -1067,6 +1067,43 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// LGTM alerts integration
+camp.route(/^\/lgtm\/alerts\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var projectId = match[1]; // eg, `g/apache/cloudstack`
+  var format = match[2];
+  var url = 'https://lgtm.com/api/v0.1/project/' + projectId + '/details';
+  var badgeData = getBadgeData('lgtm', data);
+  request(url, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      var data = JSON.parse(buffer);
+      if (data.code === 404) {
```

Comment:
Assuming when `data.code === 404` the HTTP status code is also 404, this block here and the section above where you're checking for `if (err != null)` can all be simplified to:

```js
if (checkErrorResponse(badgeData, err, res, 'project not found')) {
  sendBadge(format, badgeData);
  return;
}
```

it is a very common check, so we have a [helper](https://github.com/badges/shields/blob/3cb71037fdbed20f74a18880ed9edad4273a08c5/lib/error-helper.js) for it

Reply:
It indeed is, i'll change it